dojo/forms.py

```diff
@@ -553,6 +553,9 @@ class TestForm(forms.ModelForm):
                            required=False,
                            help_text="Add tags that help describe this test.  "
                                      "Choose from the list or add new tags.  Press TAB key to add.")
+    lead = forms.ModelChoiceField(
+	queryset=User.objects.exclude(is_staff=False),
+	required=True, label="Testing Lead")
```

Comment:
Thanks @patriknordlen great catch. Will get it fixed, and also display conditionally.